crates/ide-completion/src/item.rs

```diff
@@ -353,6 +353,7 @@ impl CompletionItem {
             relevance: CompletionRelevance::default(),
             ref_match: None,
             imports_to_add: Default::default(),
+            doc_aliases: None,
```

Comment:
I lie this appraoch! This way we can at least skip out on touching all the render methods 👍